lib/registration.pm

```diff
@@ -187,7 +187,7 @@ sub fill_in_registration_data {
     my ($addon, $uc_addon);
     fill_in_reg_server() if (!get_var("HDD_SCC_REGISTERED"));
 
-    my @known_untrusted_keys = qw(import-trusted-gpg-key-nvidia-F5113243C66B6EAE import-trusted-gpg-key-phub-9C214D4065176565);
+    my @known_untrusted_keys = qw(import-trusted-gpg-key-nvidia-F5113243C66B6EAE import-trusted-gpg-key-phub-9C214D4065176565 import-untrusted-gpg-key-ids-key-A5665AC46976A827 import-untrusted-gpg-key-idv-key-A5665AC46976A827);
```

Comment:
The tags have long name, better use this style to list them.
```
my @known_untrusted_keys = qw(
  import-trusted-gpg-key-xxxxx
  import-trusted-gpg-key-yyyyy
);
```

Reply:
Ok, updated the code following the style.